airflow/decorators/base.py

```diff
@@ -531,6 +531,9 @@ def expand(self, **kwargs: "Mappable") -> XComArg:
     def partial(self, **kwargs: Any) -> "Task[Function]":
         ...
 
+    def override(self, **kwargs: Any) -> "_TaskDecorator[Function, OperatorSubclass]":
```

Comment:
To make mypy happy now that an example DAG uses the `override()` method.

Reply:
Ugh mypy is, in fact, not happy.

Reply:
OK difference in mypy behavior is obviously from running static checks on modified files (local dev) and all files (in CI). I'll continue to investigate, but seems a little more complex to fix (although mypy is not my forté).

Reply:
Yeah I saw similar behaviour happening more frequently since we https://github.com/apache/airflow/pull/25088 - there are more cases when "small set of files changed" produced different result than "big set of files". I will investigate if we can make MyPy more "greedy" in terms of considering bigger set of files and searching "deeper" if only few files changed - because ultimately I think this is what's happening, that MyPy is not getting all the information neded to arrive to the same conclusion about parsing.

I will look a bit deeer on how MyPy makes it's decisions and maybe we can improve some of this "partial changes mypy" experience.